src/main/java/com/google/api/codegen/viewmodel/RetryCodesDefinitionView.java

```diff
@@ -18,13 +18,21 @@
 import com.google.common.collect.ImmutableSet;
 
 import io.grpc.Status.Code;
+import java.util.List;
+import javax.annotation.Nullable;
 
 @AutoValue
 public abstract class RetryCodesDefinitionView {
   public abstract String key();
 
+  @Nullable // Used in C#
+  public abstract String name();
+
   public abstract ImmutableSet<Code> codes();
 
+  @Nullable // Used in C#
+  public abstract List<String> codeNames();
```

Comment:
Looking at this case, I might just prefer to populate them (name/codeNames) for all languages and avoid the special logic for C# to open up the view class, shove these in, and rebuild it.